From a0acf1059b3d16b46977688c4f851c2234b232bb Mon Sep 17 00:00:00 2001
From: Tim Schumacher <timschumi@gmx.de>
Date: Mon, 14 May 2018 05:54:36 +0200
Subject: [PATCH] Bump Security Patch Level to 2018-05-01

Implemented Patches
===================
CVE-2017-13313  A-74114680      High        Change 214967
CVE-2017-13315  A-70721937      High
CVE-2017-13316  A-73311729      Moderate    Change 214968
CVE-2017-13319  A-71868329      High        Change 214969
CVE-2017-13323  A-73826242      Moderate    Change 214970
CVE-2017-13322  A-67862398      Moderate    Change 214972

Skipped Patches
===============
CVE-2017-13310  A-71992105      High        No ViewPager
CVE-2017-13320  A-72764648      High        No libmpeg2

Reference Version: 6.0

Change-Id: If80c7cebf54850485975fc9bb981b1f1ad6166d2
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index cea2bb3..8031045 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -81,7 +81,7 @@ ifeq "" "$(PLATFORM_SECURITY_PATCH)"
   # Can be an arbitrary string, but must be a single word.
   #
   # If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-  PLATFORM_SECURITY_PATCH := 2018-04-01
+  PLATFORM_SECURITY_PATCH := 2018-05-01
 endif
 
 ifeq "" "$(BUILD_ID)"
-- 
2.7.4

